aerleon/lib/aclgenerator.py

```diff
@@ -302,7 +304,34 @@ def __init__(self, pol, exp_info):
         self.policy = pol
         all_err = []
         all_warn = []
+        current_date = datetime.datetime.utcnow().date()
+        exp_info_date = current_date + datetime.timedelta(weeks=exp_info)
         for header, terms in pol.filters:
+            filter_name = header.FilterName(self._PLATFORM)
+            terms[:] = [
```

Comment:
What made you choose to do a list comprehension in this instance? Once list comprehensions get past a one liner they tend to become harder to read.

Reply:
Started out as a oneliner and grew after adding logging. Should I change to `filter()` then?

Reply:
They always do haha
Yea a filter would be more readable here.